.github/workflows/production-smoke-test.yml

```diff
@@ -0,0 +1,54 @@
+name: Production Build Smoke Test
+
+on:
+  push:
+    branches:
+      - master
+  workflow_dispatch:
+  pull_request:
+    branches:
+      - master
+  schedule:
+    - cron: "0 5 * * *" # Runs every day at 5am: https://docs.github.com/en/actions/reference/events-that-trigger-workflows#scheduled-events-schedule
```

Comment:
Question: Do we need the schedule, if it's run on every PR and every push to master?

Reply:
Yeah, you're probably right. I'll remove it.